sdk/src/main/java/com/xing/android/sdk/CallSpec.java

```diff
@@ -69,6 +70,21 @@ public void enqueue(Callback<RT> callback) {
         throw new UnsupportedOperationException("Not implemented yet.");
     }
 
+    public CallSpec<RT, ET> queryParam(String name, String value) {
+        builder.queryParam(name, value);
+        return this;
+    }
+
+    public CallSpec<RT, ET> formField(String name, String value) {
+        builder.formField(name, value);
+        return this;
+    }
+
+    public <U> CallSpec<RT, ET> body(Class<U> cls, U body) {
```

Comment:
There are no tests covering this... But they will come later on... with the request execution.